core/templates/dev/head/pages/community-dashboard-page/translation-opportunities/translation-opportunities.directive.ts

```diff
@@ -97,6 +120,95 @@ angular.module('oppia').directive(
             }
           };
 
+          ctrl.onClickButton = function(expId) {
+            var opportunity = getOpportunitySummary(expId);
+            $uibModal.open({
+              templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+                '/pages/community-dashboard-page/modal-templates/' +
+                'translation-modal.directive.html'),
+              backdrop: 'static',
+              size: 'lg',
```

Comment:
@NishealJ, Earlier I was using `lg` large size modal which was working fine but I think after some upgrade the `lg` flag isn't working, have we done any changes to boot-strap or angular-UI?

Reply:
Bootstrap has been upgraded from v3 to v4. Might that be relevant?

Reply:
I went through the upgrade PR and I don't see any changes related to `lg` flag in other files, are they working as expected? 

![image](https://user-images.githubusercontent.com/16653571/64493787-32c7ca80-d2a2-11e9-9843-9fa7c99683ea.png)

Reply:
Sorry, not sure. @NishealJ ... any thoughts?

@DubeySandeep it might be worth looking at the bootstrap 4 docs / angular-ui docs just to double-check what the latest spec is.

Reply:
Hi sandeep,

I’m looked into your PR i think this happens because ui-bootstap dont have a full support for bootstrap-4.

I had addressed it earlier in the below doc,
Mean while i’ll check for possible patch with ui-bootsrap
https://drive.google.com/open?id=1SGwpjbrzBr6PfsLtjShPObpZkD0vaPmYuGBxvtLPi5E

Reply:
@NishealJ Thanks for the update, I'm too looking into it, do let me know if you'll find something. Thanks!

Reply:
Is this resolved?